### DISCUSSION TITLE
keepassxc: cannot open without no3d

### DISCUSSION THREAD

**Discussion by michelesr** [netblue30/firejail#6147] [Q&A] [answered]
Without it, I'm getting the following error (in Sway WM on Arch Linux):

```
libEGL warning: wayland-egl: could not open /dev/dri/renderD128 (No such file or directory)
MESA: error: ZINK: vkEnumeratePhysicalDevices failed (VK_ERROR_INITIALIZATION_FAILED)
MESA: error: ZINK: failed to choose pdev
libEGL warning: egl: failed to create dri2 screen
```
I wonder why keepassxc needs vulkan? Seems weird to me... and I haven't updated keepassxc recently so it's strange that it suddenly requires `ignore no3d` to work.

**Comment by rusty-snake** [comment, marked as answer]
#6146

**Comment by rusty-snake** [comment]
>  I haven't updated keepassxc 

But mesa. i.e. libraries it uses.

> I wonder why keepassxc needs vulkan?

HW accelerated rendering.

Actually it should not need but want it. And fallback to cpu/software rendering.

**Comment by michelesr** [comment]
> HW accelerated rendering.

Out of curiosity, is it to decrypt the master key, or other unrelated purposes?

**Reply by rusty-snake** [reply]
Qt
